### PR TITLE
Enable website display for mobile devices

### PR DESCRIPTION
**Pull request by aniluede**
Breadcrumbs, toolbar and footer changes, and css files improvements

### CODE REVIEW

File: src/css/header.css [hunk 1]
```diff
@@ -283,3 +283,449 @@ body {
...
+
+.sections-navbar-dropdown.is-right {
+  left: auto;
+  right: 0;
+}
+
+.sections-navbar-dropdown a.navbar-item:hover {
+  background: var(--navbar-menu_hover-background);
+}
+/* .sections-navbar-link::after {
```
Comment (michelu89) [inline src/css/header.css:595]: Can this comment be deleted?

File: src/css/search.css [hunk 2]
```diff
@@ -168,3 +84,134 @@
...
+    padding-right: 2rem;
+  }
+
+  .search-result-dropdown-menu {
+    margin-left: 2.5rem !important;
+    margin-top: -1.5rem !important;
+  }
+}
+
+/* @media screen and (min-width: 769px) {
```
Comment (michelu89) [inline src/css/search.css:181]: Can this comment be deleted?

File: src/css/sections.css [hunk 3]
```diff
@@ -265,60 +253,186 @@
...
 .sectionBlue .link > p > a,
 .sectionPurple .link > p > a {
   color: var(--color-white);
-  font-weight: bold;
+  font-weight: 500;
+}
+
+/* Tiles to sections css */
+
+/* .sectionPurple .innerTile,
```
Comment (michelu89) [inline src/css/sections.css:267]: Can this comment be deleted?

File: src/css/sections.css [hunk 4]
```diff
@@ -265,60 +253,186 @@
...
+.innerTile > div > div.imageblock.xref-page > div > a > img {
+  margin: 1rem 0.25rem !important;
+  height: 6.5rem !important;
 }
 
-@media screen and (max-width: 1140px) {
-  /* .tile {
-    width: calc((100% / 2) - 1rem);
-  } */
+/* .innerTile svg {
```
Comment (michelu89) [inline src/css/sections.css:285]: Can this comment be deleted?

File: src/partials/sections-header-content.hbs [hunk 5]
```diff
@@ -22,11 +22,13 @@
               </g>
             </g>
           </svg>
+          {{!-- USED IN MOBILE RES --}}
           {{!-- <span class="sections-docu-title">
```
Comment (michelu89) [inline src/partials/sections-header-content.hbs:26]: Can this comment be deleted?

**Review by michelu89 — APPROVED** [review]

File: src/css/header.css (hunk 1 above)
Comment (aniluede) [reply src/css/header.css:595]: Not yet.

File: src/css/search.css (hunk 2 above)
Comment (aniluede) [reply src/css/search.css:181]: Not yet.

File: src/css/sections.css (hunk 3 above)
Comment (aniluede) [reply src/css/sections.css:267]: Not yet.

File: src/css/sections.css (hunk 4 above)
Comment (aniluede) [reply src/css/sections.css:285]: Not yet.

File: src/partials/sections-header-content.hbs (hunk 5 above)
Comment (aniluede) [reply src/partials/sections-header-content.hbs:26]: Not yet.

**Review by atextor — APPROVED** [review]

**Pull request merged**